config/etc/system-image/writable-paths

```diff
@@ -109,4 +109,6 @@
 /etc/default/swapfile                   auto                    persistent  transition  none
 # allow system wide proxy setting
 /etc/environment                        auto                    persistent  transition  none
+# https://github.com/snapcore/snapd/pull/3866
+/var/cache/snapd                         auto                   synced      none        none
```

Comment:
Why is this synced? Will we ship things in there (probably sensibel)?

Reply:
I dont think there will be anything in the squashfs initially ... @chipaca can probably elaborate why "synced" should be used...

Reply:
so that if we _want_ to ship the caches pre-seeded, we can.

Reply:
you are aware that chaches with the same filename in both places will not be synced i assume ?

Reply:
that's exactly the behaviour I want, yes, thank you.

We're leaving ourselves the option of shipping a default cache that would be used if the snapd on the device doesn't get network for a while, say. E.g. if it's only rarely online and has been updated out-of-band.

If the file exists it'll probably be fresher / better than the seeded one.